webui/react/src/Analytics.ts

```diff
@@ -48,21 +48,27 @@ const getReadyAnalytics = (): SegmentAnalytics.AnalyticsJS | undefined => {
   return undefined;
 };
 
-export const setupAnalytics = async (info: DeterminedInfo): Promise<void> => {
+export const setupAnalytics = async (auth: Auth, info: DeterminedInfo): Promise<void> => {
   if (!data.analytics) data.analytics = getReadyAnalytics();
   if (!data.analytics || data.isEnabled) return;
 
   /*
    * Segment key should be 32 characters composed of upper case letters,
    * lower case letters and numbers 0-9.
    */
-  if (info.isTelemetryEnabled) {
+  if (auth.user && info.isTelemetryEnabled) {
     try {
       const telemetry = await getTelemetry({});
       const isProperKey = telemetry.segmentKey && /^[a-z0-9]{32}$/i.test(telemetry.segmentKey);
       if (isProperKey) {
         data.analytics.load(telemetry.segmentKey || '');
-        data.analytics.identify(info.clusterId);
+        data.analytics.identify(auth.user.username, {
```

Comment:
the first argument to the identify call seems to be used as userId. If we want to to be unique  by itself maybe we should merge it with clusterId? eg `username@clusterid`. Otherwise we'll probably get a lot of similar usernames; most likely `determined`

Reply:
modified this portion to use `user id` as the identify target (instead of username or cluster id)